packages/fxa-settings/src/pages/Signin/index.test.tsx

```diff
@@ -601,205 +605,248 @@ describe('Signin', () => {
         expect(GleanMetrics.login.error).toHaveBeenCalledTimes(1);
       });
     });
-  });
 
-  describe('user does not have a password', () => {
-    it('renders as expected without linked account', () => {
-      render({ hasPassword: false });
+    describe('user does not have a password', () => {
+      it('renders as expected without linked account', () => {
+        render({ hasPassword: false });
 
-      signInHeaderRendered();
-      avatarAndEmailRendered();
-      signInButtonAndSeparatorRendered();
-      thirdPartyAuthRendered();
-      privacyAndTermsRendered();
-      differentAccountLinkRendered();
-      resetPasswordLinkRendered();
+        signInHeaderRendered();
+        defaultAvatarAndEmailRendered();
+        signInButtonAndSeparatorRendered();
+        thirdPartyAuthRendered();
+        privacyAndTermsRendered();
+        differentAccountLinkRendered();
+        resetPasswordLinkRendered();
 
-      passwordInputNotRendered();
+        passwordInputNotRendered();
+      });
+
+      it('renders as expected with linked account', () => {
+        render({ hasPassword: false, hasLinkedAccount: true });
+        signInHeaderRendered();
+        defaultAvatarAndEmailRendered();
+        thirdPartyAuthRendered();
+        privacyAndTermsRendered();
+
+        passwordInputNotRendered();
+        expect(
+          screen.queryByRole('link', { name: 'Forgot password?' })
+        ).not.toBeInTheDocument();
+        expect(screen.queryByText('Or')).not.toBeInTheDocument();
+        expect(
+          screen.queryByRole('button', { name: 'Sign in' })
+        ).not.toBeInTheDocument();
+      });
     });
+  });
 
-    it('renders as expected with linked account', () => {
-      render({ hasPassword: false, hasLinkedAccount: true });
+  describe('with sessionToken', () => {
```

Comment:
Sorry about the big change set, I noticed that the `with session token` scenario was not nested within the higher level describe block for the sign in tests. Fixing that makes the change set look bigger than it actually is. 

The main changes include checking for default avatar if there is no session token, and testing the "user does not have a password" scenario for both with and without session token.